src/user.ts

```diff
@@ -191,7 +202,33 @@ export class UserApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
-}
-
 
+  /*
+   * Function to a User token, status, reason code, and channel, send
+   * those to Marqeta, and transition a User account between states.
+   */
+  async transition(status: Partial<Transition>): Promise<{
+    success: boolean,
+    body?: Transition,
+    error?: MarqetaError,
+  }> {
```

Comment:
I believe you that this is the way it is... but it seems exceptionally odd that this isn't a User and some stuff...

Reply:
https://www.marqeta.com/docs/core-api/user-transitions

Reply:
I thought about creating a `user-transitions.ts` module, but figured that was redundant.

Reply:
Yeah, this is the better route... but it is very odd of them...